libs/dh/shared/data-access-msw/src/lib/mocks/marketParticipant.ts

```diff
@@ -45,6 +47,15 @@ function getOrganizations(apiBase: string) {
   );
 }
 
+function getAllOrganizationsWithActors(apiBase: string) {
+  return rest.get(
+    `${apiBase}/v1/MarketParticipant/Organization/GetAllOrganizationsWithActors`,
+    (req, res, ctx) => {
+      return res(ctx.json(organizationWithActorsData));
```

Comment:
suggestion: can we convert `organizationWithActorsData` to a TS variable? That way it can be typed, so changes to the endpoint will not result in out of sync mock.

Reply:
Does that work with none graphql as well?